workflows/funcscan.nf

```diff
@@ -5,8 +5,6 @@
 */
 include { validateParameters; paramsHelp; paramsSummaryLog; paramsSummaryMap; fromSamplesheet } from 'plugin/nf-validation'
 
-include { paramsSummaryLog; paramsSummaryMap; fromSamplesheet } from 'plugin/nf-validation'
```

Comment:
I wouldn't remove, just comment out. We will switch to this in the future (if e haven't already 🤔